src/interfaces/wallet.cpp

```diff
@@ -202,6 +202,12 @@ class WalletImpl : public Wallet
         LOCK(m_wallet->cs_wallet);
         return m_wallet->ListLockedCoins(outputs);
     }
+    bool IsUsedDestination(const CTxDestination& dst) override
+    {   
+        auto locked_chain = m_wallet->chain().lock();
```

Comment:
Why are you locking this?

Reply:
sorry, think i misunderstood what this line did, will remove

Reply:
this has now been removed